src/main/java/net/syscon/elite/service/BookingService.java

```diff
@@ -305,7 +319,7 @@ private long daysSinceDetailBecameEffective(final PrivilegeDetail currentDetail)
     }
 
     @VerifyBookingAccess
-    public Page<ScheduledEvent> getBookingActivities(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final long offset, final long limit, final String orderByFields, final Order order) {
+    public net.syscon.elite.api.support.Page<ScheduledEvent> getBookingActivities(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final long offset, final long limit, final String orderByFields, final Order order) {
```

Comment:
I'm not 100% sure this is the best way to tackle this, but the Page implementation above conflicts with the Page implementation used for Pageable and JPA repository. They can't both me imported and Java doesn't allow renaming of imports. Happy for suggestions on how to tackle this better.

Reply:
I don't think there's much that you can do about this, you just end up having to pick which one to use the full package name for 👍